src/PokeContainer.js

```diff
@@ -0,0 +1,149 @@
+import React, { Component } from "react";
+import Pokelist from "./Pokelist.js";
+import Dropdown from "./Dropdown.js";
+import criteriaList from "./criteriaList.js";
+import typeList from "./typeList.js";
+import Loader from "react-loader-spinner";
+
+class PokeContainer extends Component {
+  state = {
+    pokeData: [],
+    loading: true,
+    query: null,
+    sortCriteria: "",
+    sortOrder: "asc",
+    sortType: "",
+    page: 1,
+    lastPage: 1,
+  };
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  fetchData = async () => {
+    const { query, sortCriteria, sortOrder, sortType } = this.state;
+    let url = "https://pokedex-alchemy.herokuapp.com/api/pokedex";
+    let searchParams = new URLSearchParams();
+
+    searchParams.set("page", this.state.page);
```

Comment:
Not resetting the page in state back to 1 can cause issues if the user is on a different page then searches. They won't see their results, but will see a blank screen instead because they have to go back to page 1 manually.